app/routes/queries/bql.js

```diff
@@ -0,0 +1,28 @@
+/*
+ *  Copyright 2020, Yahoo Inc.
+ *  Licensed under the terms of the Apache License, Version 2.0.
+ *  See the LICENSE file associated with the project for terms.
+ */
+import { inject as service } from '@ember/service';
+import DefaultQueryRoute from 'bullet-ui/routes/queries/default-query';
+
+const DEFAULT_QUERY = 'SELECT * FROM STREAM(20000, TIME) LIMIT 1 ;';
```

Comment:
Not necessary but I've been hard coding BQL with `;` at the end. Used space because at some point the recreation logic in the ui used to include a semicolon and since limit was optional, it would show up with an empty space. Do you want to remove the entire space semi?

Reply:
Nope, don't really care. Just thought it looked odd